resource-aggregate/.plantuml

```diff
@@ -237,4 +237,57 @@ deactivate Gateway
 ResourceAggregate --> Bus : DeviceMetadataUpdated
 deactivate ResourceAggregate
 deactivate Server
+@enduml
+
+@startuml
+title "Cancel resource command - similar for all Update/Retrieve/Create/Delete commands"
+
+participant "Resource Aggregate" as ResourceAggregate
+control "Event Bus" as Bus
+entity "Client" as Client
+entity "Client1" as Client1
```

Comment:
I would remove Client1 from diagrams. It doesn't clarify anything. It's visible that events are sent to eventBus and that's enough. Same for second cancelation diagram.

Reply:
entity "OCF Server" as Server